model/src/main/java/org/cbioportal/model/OncoKBDataCountItem.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class OncoKBDataCountItem implements Serializable {
+
+    public enum OncoKBDataType {
```

Comment:
We don't need data type here. oncokb will always on sample since the annotation is on mutation level